src/components/views/Chart.js

```diff
@@ -11,6 +18,13 @@ export default props => {
 
     const significantDigits = 5;
 
+    function ordinalToActualValue(ordinal, datasetIndex = 0) {
+        const currentDataPoint = props.data.datasets[datasetIndex].data.find(e => e.x === Number(ordinal));
+        if (currentDataPoint) {
+            return currentDataPoint.xActualValue;
```

Comment:
'Label' might be as descriptive as 'ActualValue'. If I am commenting on this it's because the code itself looks fine 😬

Reply:
We think alike! I actually called this `xLabel` initially. As it turns out, the tooltip callback for Chart.js passes an object with a property named `xLabel` which means something totally different (see [this line](https://github.com/mozilla/firefox-test-tube/pull/66/files#diff-acdb4fa6fe657229ba882e2a12c6b585R42)), so it was getting confusing.